examples/dtls-wolfssl/README.md

```diff
@@ -0,0 +1,35 @@
+# dtls_wolfssl example
+
+This example shows how to use DTLS with wolfSSL
+
+## SOCK vs. Socket
+
+This example is configured to use socks instead of sockets (over GNRC).
+It's possible to use sockets, which give a more similar approach to the
+UNIX version of wolfSSL.
+
+## Fast configuration (Between RIOT instances):
+
+Preparing the logical interfaces:
+
+    ./../../dist/tools/tapsetup/tapsetup --create 2
+
+For the server instance:
```

Comment:
These steps are not in sync with with the PR description which contains the working version.